docs/devicemanager.apib

```diff
@@ -529,6 +530,18 @@ In this example, there is only one template (ID 1).
                     "total": 1
                 }
             }
+
```

Comment:
for reference: this ends up rendered as the following

![image](https://user-images.githubusercontent.com/25080094/36731255-f3c829e4-1ba8-11e8-8a2e-e07e847abca6.png)